models/User.js

```diff
@@ -5,9 +5,9 @@ const userSchema = new Schema({
 	isOwner: Boolean,
 	isPlayer: Boolean,
 	isAdmin: Boolean,
-	ign: String,
 	elo: Number,
```

Comment:
I liked the idea of the User possessing an ELO based off of some possible algorithm (their highest player or a weighted average etc.) to give a quick reference, but if we're storing an array of player objects inside of them I don't think the queries will be a problem or slow.